api/types/trust.go

```diff
@@ -57,9 +59,19 @@ var CertAuthTypes = []CertAuthType{HostCA, UserCA, DatabaseCA, OpenSSHCA, JWTSig
 // major version, so that we can avoid erroring out when a potentially older
 // remote server doesn't know about them.
 func (c CertAuthType) NewlyAdded() bool {
+	return c.addedInVer().Major == semver.New(teleport.Version).Major
```

Comment:
Do we guarantee that `teleport.Version` is always a semver-valid string?

I guess we'd figure that out pretty quickly anyway 😅

Reply:
Now that I think about it, we should have a package-scope variable for `semver.New(teleport.Version)` so we avoid parsing the string every time and we immediately panic at package import rather than potentially not having a test that invokes `NewlyAdded` in `api`.